<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" idbasedtr="true">
 <author>懒人智能</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font: 75 10pt &quot;微软雅黑&quot;;}

#lb_title{
color:red;
	font: 75 11pt &quot;微软雅黑&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>UI加载与初始化</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>参数:必须遵守命名规则 ui生成的py文件都对应UI_class参数的值</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>类主体:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="bt_copy_class">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>复制主体</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2" colspan="2">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameLineEdit">
            <property name="text">
             <string>window_main</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="classLabel">
            <property name="text">
             <string>Class:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cbb_class">
            <item>
             <property name="text">
              <string>QMainWindow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>QWidget</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>QDialog</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="uiClassLabel">
            <property name="text">
             <string>ui_path:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="ed_ui_path">
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string notr="true"/>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="bt_ui_path">
            <property name="text">
             <string>1.选择designer生成的py文件</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string>Title:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="titleLineEdit">
            <property name="text">
             <string>我的主窗口</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="sizeLabel">
            <property name="text">
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="sizeLineEdit">
            <property name="text">
             <string>500*500</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="moveLabel">
            <property name="text">
             <string>Move:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="moveLineEdit">
            <property name="text">
             <string>200,200</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="flagsLabel">
            <property name="text">
             <string>Flags:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="2">
             <widget class="QCheckBox" name="cb_wbk">
              <property name="text">
               <string>窗口无边框</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="cb_max_bt">
              <property name="tabletTracking">
               <bool>false</bool>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>最大化按钮</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="cb_min_bt">
              <property name="text">
               <string>最小化按钮</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="3">
             <widget class="QComboBox" name="cbb_flags">
              <item>
               <property name="text">
                <string>Qt.Widget默认值 自动识别是什么窗口</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qt.Window 部件始终为窗口</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qt.Dialog 部件为对话框</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qt.Sheet 部件是一个Macintosh表单</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qt.Drawer 部件是一个Macintosh抽屉</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qt.Popup 部件是一个弹出式顶层窗口</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qt.Tool 部件是一个工具窗口</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qt.ToolTip:部件是一个提示窗口</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qt.Desktop 部件是一个桌面</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qt.SplashScreen 部件是一个欢迎窗口</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qt.SubWindow:部件是一个子窗口</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QCheckBox" name="cb_exit">
              <property name="text">
               <string>关闭按钮</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="cb_top">
              <property name="text">
               <string>永远置顶</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="cb_bjtm">
              <property name="text">
               <string>背景为透明</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="ed_transp">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>窗口透明度(0-1之间的小数):</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="8" column="1">
           <widget class="QPushButton" name="bt_code_generation">
            <property name="text">
             <string>2.生成代码与预览</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QTextEdit" name="ed_code_class">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:10pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" rowspan="2">
         <widget class="QPushButton" name="bt_copy_init">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>复制实例</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>模块引用:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" rowspan="2">
         <widget class="QPushButton" name="bt_copy_import">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>复制引用</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>实例化:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" rowspan="2" colspan="2">
         <widget class="QTextEdit" name="ed_code_init">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:10pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QTextEdit" name="ed_code_import">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:10pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>3.信号与槽(预览后来这里)</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lb_title">
          <property name="text">
           <string>当前选中的控件为:bt_xxx(QPushButton)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="signalLabel_2">
            <property name="text">
             <string>常用Signal:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbb_signal"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="signalLabel_3">
            <property name="text">
             <string>重写QEvent:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cbb_qevent"/>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="bt_pyqtapi">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>PYqt5官方api文档</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bt_qss">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>官方QSS文档英文版</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bt_qss_2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>QSS中文帮助文档</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>信号绑定槽函数:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_copy_bind">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>复制绑定</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="ed_signal"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>槽函数/重写 实现代码:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_copy_connect">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>复制实现</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="ed_connect"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
